docs/core/compatibility/2.1.md

```diff
@@ -12,10 +12,18 @@ If you're migrating to version 2.1 of .NET Core, the breaking changes listed in
 - [Private fields added to built-in struct types](#private-fields-added-to-built-in-struct-types)
 - [OpenSSL versions on macOS](#openssl-versions-on-macos)
 
-[!INCLUDE[Private fields added to built-in struct types](~/includes/core-changes/corefx/2.1/instantiate-struct.md)]
+[!INCLUDE[Private fields added to built-in struct types](../../../includes/core-changes/corefx/2.1/instantiate-struct.md)]
 
 ***
 
 [!INCLUDE [OpenSSL versions on macOS](../../../includes/core-changes/corefx/openssl-dependencies-macos.md)]
 
 ***
+
+## MSBuild
+
+- [Project tools now included in SDK](#project-tools-now-included-in-sdk)
+
+[!INCLUDE [DotNetCliToolReference project elements removed for bundled tools](../../../includes/core-changes/msbuild/2.1/dotnetclitoolreference.md)]
```

Comment:
PLease also add it to https://github.com/dotnet/docs/blob/master/docs/core/compatibility/msbuild.md. (And update the version range in the title of that file.)

Reply:
That article seems to be 3.0-3.1 focused. Should there be another article or just another header that ranges 2.0-2.1?

Reply:
It focuses on those versions because up until now, there were no MSBuild breaking changes prior to .NET 3.0. So you just need to update the starting version from 3.0 to 2.0.